lib/services/bitbucket_commit_hook.rb

```diff
@@ -30,16 +30,21 @@ def receive_webhook
 
   def comment_on_record(ref_num, commit)
     record_type = ref_num =~ /-[0-9]+-/ ? "requirements" : "features"
+    
+    begin
+      email = Mail::Address.new(commit.author.raw)
```

Comment:
is `author` ever nil?

Reply:
If `author` is nil, Bitbucket has changed their hook and we'd have a lot of issues to deal with.